portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,72 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+a.button {
```

Comment:
This seems to be a duplicate CSS of the `infiniterun/gamestyle.css` above? Since there doesn't seem to be a ton of CSS right now you can also consider putting everything in one file, but up to you.